src/components/DatePicker/index.ios.js

```diff
@@ -19,18 +20,29 @@ const datepickerPropTypes = {
 class Datepicker extends React.Component {
     constructor(props) {
         super(props);
-
+        const value = DateUtils.getDateAsText(props.value) || DateUtils.getDateAsText(props.defaultValue) || CONST.DATE.MOMENT_FORMAT_STRING;
```

Comment:
How come we are doing this in the constructor for `index.ios.js` but android doesn't have it?

Reply:
This is the initialization value for the the state `value`. We aren't doing it for android because we don't need to handle selected date and value separately for android.